packages/bpk-component-modal/src/BpkModalV2/BpkModal.tsx

```diff
@@ -41,32 +44,56 @@ export type Props = {
   showHeader?: boolean;
   title?: string | null;
   wide?: boolean;
+  modalStyle?: ModalStyle;
 };
 
 const Header = ({
   closeLabel,
   id,
+  modalStyle,
   onClose,
   title,
 }: {
   closeLabel: string;
   id: string | undefined;
+  modalStyle: string;
   onClose: () => void | null;
   title?: string | null;
 }) => {
   if (title) {
     return (
-      <div id={id} className={getClassName('bpk-modal__header-title')}>
+      <div
+        id={id}
+        className={[
+          getClassName('bpk-modal__header-title'),
+          getClassName(`bpk-modal__header-title-style--${modalStyle}`),
+        ].join(' ')}
+      >
         <div className={getClassName('bpk-modal__header-title-container')}>
           <Heading>{title}</Heading>
         </div>
-        <BpkCloseButton label={closeLabel} onClick={onClose} />
+        <BpkCloseButton
+          className={getClassName(
+            `bpk-modal__close-button-style--${modalStyle}`,
+          )}
+          label={closeLabel}
+          onClick={onClose}
+        />
       </div>
     );
   }
   return (
-    <div className={getClassName('bpk-modal__button-container')}>
-      <BpkCloseButton label={closeLabel} onClick={onClose} />
+    <div
+      className={[
+        getClassName('bpk-modal__button-container'),
+        getClassName(`bpk-modal__button-container-style--${modalStyle}`),
+      ].join(' ')}
+    >
+      <BpkCloseButton
+        className={getClassName(`bpk-modal__close-button-style--${modalStyle}`)}
```

Comment:
We shouldn't provide `className` to backpack components, extending `className` when it is already being passed is acceptable. However, adding new instances is not. For reference, this is related to the work that web-enablement is taking on

Reply:
Is this an FYI? If it isn't, what should I be doing? @artemzhydkov96 has already approved it and squashed and merged it.

Reply:
👋 Hey @ameliajfitzpatrick - not to worry. More of an FYI. There is no action for you, I've passed this onto the team so that they are aware for future reviews.